python_modules/dagster/dagster/_core/storage/state/state_versions.py

```diff
@@ -0,0 +1,36 @@
+from collections.abc import Mapping
+from typing import Optional
+
+from dagster_shared.record import record
+from dagster_shared.serdes import whitelist_for_serdes
+
+from dagster._time import get_current_timestamp
+
+
+@whitelist_for_serdes
+@record
+class StateVersionInfo:
```

Comment:
if we rename the instance component, maybe these should sync up but they are probably fine